src/Components/Components/src/RenderTree/Renderer.cs

```diff
@@ -369,6 +371,16 @@ internal void TrackReplacedEventHandlerId(ulong oldEventHandlerId, ulong newEven
             _eventHandlerIdReplacements.Add(oldEventHandlerId, newEventHandlerId);
         }
 
+        /// <summary>
+        /// Gets the <see cref="IJSRuntime"/> service from the <see cref="Renderer"/>'s service provider.
```

Comment:
We generally do not document internal APIs. But if you would like to, I'd leave it at 
```suggestion
        /// Gets the <see cref="IJSRuntime"/> instance.
```

The implementation details aren't useful to consumers and might change which would leave an out-of-date doc